### PR TITLE
Display images of event venues

### PR DESCRIPTION
**Pull request by ethax-ross**
### Context

Some event buildings will have an `image_url` set; if they do we want to surface this image below the Google Map of the venue location.

### Changes proposed in this pull request

- Display images of event venues
- Clean out some unused event CSS rules
- Add a link to open the map in Google Maps

### Guidance to review

The `show` template could use an overhaul in general; I'm going to create a ticket to tidy it up.

I think there is an issue with our local Google Maps API key not allowing localhost (or review apps, for that matter). You can preview the change [here](https://review-get-into-teaching-app-1526.london.cloudapps.digital/events/train-to-teach-birmingham-event-110921)

| Mobile      | Desktop |
| ----------- | ----------- |
| Screenshot 2021-07-21 at 14 41 31      | Screenshot 2021-07-21 at 14 42 16       |

### CODE REVIEW

File: spec/factories/api/event_building_api_factory.rb [hunk 1]
```diff
@@ -7,5 +7,6 @@
     address_line3 { nil }
     address_city { "Manchester" }
     address_postcode { "MA1 1AM" }
+    image_url { "http:://example.com/image.png" }
```
Comment (peteryates) [inline spec/factories/api/event_building_api_factory.rb:10]:
```suggestion
    image_url { "http://example.com/image.png" }
```

Comment (ethax-ross) [reply spec/factories/api/event_building_api_factory.rb:10]: Whoops, ta

File: spec/factories/internal/event_building_factory.rb [hunk 2]
```diff
@@ -7,5 +7,6 @@
     address_line3 { "test" }
     address_city { "test" }
     address_postcode { "M1 7AX" }
+    image_url { "http:://example.com/image.png" }
```
Comment (peteryates) [inline spec/factories/internal/event_building_factory.rb:10]:
```suggestion
    image_url { "http://example.com/image.png" }
```

**Review by peteryates — COMMENTED** [review]
Looks good to me!

**Review by peteryates — APPROVED** [review]

**Pull request merged**